### ISSUE TITLE
Insufficient colour contrast

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#1442]
## Needs
- [ ] New visual design

## Acceptance Criteria
- [ ] All colour combos pass contrast checks for WCAG 2.0 AA

See #606 and also these fails from El R (confirmed by @gestchild )

Image search:
<img width="202" alt="results" src="https://user-images.githubusercontent.com/1446139/29962787-4635beba-8efc-11e7-9b87-1ca80491d675.png">

#717171 on #F0EDE3

In the footer:
<img width="181" alt="footer" src="https://user-images.githubusercontent.com/1446139/29962795-4f1a6f9e-8efc-11e7-9777-64e06e6c0f35.png">

#0aa38f on #000000

And still on the explore section:
<img width="84" alt="explore" src="https://user-images.githubusercontent.com/1446139/29962815-6915fa3a-8efc-11e7-897d-9e0a4701e67d.png">

#0aa38f on #F0EDE3

**Comment by gestchild** [comment]
Actually, #0aa38f on #000000 is fine it passes with a colour contrast ratio of 6.65:1.

However, the other 2 do fail

**Comment by davidpmccormick** [comment]
@jennpb in terms of these two:

![](https://user-images.githubusercontent.com/1446139/29962815-6915fa3a-8efc-11e7-897d-9e0a4701e67d.png)

![](https://user-images.githubusercontent.com/1446139/29962787-4635beba-8efc-11e7-9b87-1ca80491d675.png)

to get enough contrast, we can change the top one from 'mint' to 'turquoise-text' and the bottom one from 'pewter' to 'charcoal' (https://cardigan.wellcomecollection.org/components/detail/palette.html)

Is that acceptable? Looks good to me, but then I've got defective eyes.

**Comment by jennpb** [comment]
Does elf-green pass the check? If so, can we try that one instead?

Moving from pewter to charcoal on the second problem seems a good idea for now, but we can revisit with @Heesoomoon and @Norvard

**Comment by davidpmccormick** [comment]
> Does elf-green pass the check?

'fraid not.

**Comment by jennpb** [comment]
:-1: 

@Heesoomoon do you think we should change those labels to turquoise as DM mentions above? The label is not a link anyway, so I don't see a problem with changing it to meet accessibility requirements.

**Comment by Heesoomoon** [comment]
For now, 'Charcoal' might be better option if the goal of this is to improve accessibility and to differentiate from a link. When Tomi gets back, I would like to review the color palette and address the accessibility issues in more holistic ways.

**Comment by Heesoomoon** [comment]
Btw, does the colour of the promo description over 'Cream' background pass the test? It's quite subtle on my Mac laptop.
![feature](https://user-images.githubusercontent.com/29206031/30130862-90cc03ae-9342-11e7-90d3-91c9174c8e6d.png)

**Comment by gestchild** [comment]
No, it's the same as the colours as the '35 results for...' text

**Comment by Heesoomoon** [comment]
Let's change to charcoal for now. I'll suggest few options to Tomi and share the updated typography spec next week.
